### ISSUE TITLE
After fresh install, Symata throw me an error

### ISSUE THREAD

**Issue by BuddhiLW** [jlapeyre/Symata.jl#199]
I'm unable to use Symata, the following occurs

```jl
julia> using Symata
ERROR: InitError: KeyError: key :containers not found
Stacktrace:
 [1] __getproperty(::PyCall.PyObject, ::Symbol) at /home/buddhilw/.julia/packages/PyCall/BcTLp/src/PyCall.jl:307
 [2] getproperty at /home/buddhilw/.julia/packages/PyCall/BcTLp/src/PyCall.jl:312 [inlined]
 [3] getindex at /home/buddhilw/.julia/packages/PyCall/BcTLp/src/PyCall.jl:343 [inlined]
 [4] populate_py_to_mx_dict() at /home/buddhilw/.julia/packages/Symata/1oDeR/src/sympy.jl:243
 [5] init_sympy() at /home/buddhilw/.julia/packages/Symata/1oDeR/src/sympy.jl:852
 [6] do_init() at /home/buddhilw/.julia/packages/Symata/1oDeR/src/Symata.jl:130
 [7] __init__() at /home/buddhilw/.julia/packages/Symata/1oDeR/src/Symata.jl:124
 [8] run_repl(::REPL.AbstractREPL, ::Any) at /build/julia/src/julia-1.5.3/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:288
during initialization of module Symata
```

julia> versioninfo()
Julia Version 1.5.3
Commit 788b2c77c1* (2020-11-09 13:37 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-10.0.1 (ORCJIT, skylake)

**Comment by deoradh** [comment]
Any progress?  I'm encountering this as well.

`(@v1.5) pkg> st
Status `C:\Users\kfarm\.julia\environments\v1.5\Project.toml`
  [7073ff75] IJulia v1.23.2
  [91a5bcdd] Plots v1.11.0
  [438e738f] PyCall v1.92.2
  [123dc426] SymEngine v0.8.3
  [a906b1d5] Symata v0.4.5

julia> using Symata
[ Info: Precompiling Symata [a906b1d5-d016-55c4-aab3-8a20cba0db2a]
WARNING: could not import Base.Test into Symata
WARNING: could not import Base.search into Symata
WARNING: could not import Base.next into Symata
ERROR: InitError: KeyError: key :containers not found
Stacktrace:
 [1] __getproperty(::PyCall.PyObject, ::Symbol) at C:\Users\kfarm\.julia\packages\PyCall\tqyST\src\PyCall.jl:307
 [2] getproperty at C:\Users\kfarm\.julia\packages\PyCall\tqyST\src\PyCall.jl:312 [inlined]
 [3] getindex at C:\Users\kfarm\.julia\packages\PyCall\tqyST\src\PyCall.jl:343 [inlined]
 [4] populate_py_to_mx_dict() at C:\Users\kfarm\.julia\packages\Symata\1oDeR\src\sympy.jl:243
 [5] init_sympy() at C:\Users\kfarm\.julia\packages\Symata\1oDeR\src\sympy.jl:852
 [6] do_init() at C:\Users\kfarm\.julia\packages\Symata\1oDeR\src\Symata.jl:130
 [7] __init__() at C:\Users\kfarm\.julia\packages\Symata\1oDeR\src\Symata.jl:124
 [8] _include_from_serialized(::String, ::Array{Any,1}) at .\loading.jl:697
 [9] _require_from_serialized(::String) at .\loading.jl:749
 [10] _require(::Base.PkgId) at .\loading.jl:1040
 [11] require(::Base.PkgId) at .\loading.jl:928
 [12] require(::Module, ::Symbol) at .\loading.jl:923
during initialization of module Symata`

**Comment by BuddhiLW** [comment]
Unfortunately, no. I just gave up using it :/

I won't be trying it soon, again. I was going to use it on a research. But, life took turns, you know?

Good luck, buddy!

**Comment by jlapeyre** [comment]
Thanks for raising the issue. I think Symata no longer runs with recent versions of Julia. It should not be too difficult to fix, but requires some investment in time.

**Comment by jlapeyre** [comment]
This bug has been fixed in v0.4.9
